src/connectors/clear-refinements/connectClearRefinements.js

```diff
@@ -15,81 +15,81 @@ var customClearAll = connectClearAll(function render(params, isFirstRendering) {
+ * is set to `true`, `refine` will also clear the query and `excludedAttributes` can
  * prevent certain attributes from being cleared.
  *
  * @type {Connector}
- * @param {function(ClearAllRenderingOptions, boolean)} renderFn Rendering function for the custom **ClearAll** widget.
+ * @param {function(ClearRefinementsRenderingOptions, boolean)} renderFn Rendering function for the custom **ClearRefinements** widget.
  * @param {function} unmountFn Unmount function called when the widget is disposed.
- * @return {function(CustomClearAllWidgetOptions)} Re-usable widget factory for a custom **ClearAll** widget.
+ * @return {function(CustomClearRefinementsWidgetOptions)} Re-usable widget factory for a custom **ClearRefinements** widget.
  * @example
- * // custom `renderFn` to render the custom ClearAll widget
- * function renderFn(ClearAllRenderingOptions, isFirstRendering) {
- *   var containerNode = ClearAllRenderingOptions.widgetParams.containerNode;
+ * // custom `renderFn` to render the custom ClearRefinements widget
+ * function renderFn(ClearRefinementsRenderingOptions, isFirstRendering) {
+ *   var containerNode = ClearRefinementsRenderingOptions.widgetParams.containerNode;
  *   if (isFirstRendering) {
  *     var markup = $('<button id="custom-clear-all">Clear All</button>');
  *     containerNode.append(markup);
  *
  *     markup.on('click', function(event) {
  *       event.preventDefault();
- *       ClearAllRenderingOptions.refine();
+ *       ClearRefinementsRenderingOptions.refine();
  *     })
  *   }
  *
- *   var clearAllCTA = containerNode.find('#custom-clear-all');
- *   clearAllCTA.attr('disabled', !ClearAllRenderingOptions.hasRefinements)
+ *   var clearRefinementsCTA = containerNode.find('#custom-clear-all');
+ *   clearRefinementsCTA.attr('disabled', !ClearRefinementsRenderingOptions.hasRefinements)
  * };
  *
- * // connect `renderFn` to ClearAll logic
- * var customClearAllWidget = instantsearch.connectors.connectClearAll(renderFn);
+ * // connect `renderFn` to ClearRefinements logic
+ * var customClearRefinementsWidget = instantsearch.connectors.connectClearRefinements(renderFn);
  *
  * // mount widget on the page
  * search.addWidget(
- *   customClearAllWidget({
+ *   customClearRefinementsWidget({
  *     containerNode: $('#custom-clear-all-container'),
  *   })
  * );
  */
-export default function connectClearAll(renderFn, unmountFn) {
+export default function connectClearRefinements(renderFn, unmountFn) {
   checkRendering(renderFn, usage);
 
   return (widgetParams = {}) => {
-    const { excludeAttributes = [], clearsQuery = false } = widgetParams;
+    const { excludedAttributes = [], clearsQuery = false } = widgetParams;
 
     return {
       init({ helper, instantSearchInstance, createURL }) {
         const attributesToClear = getAttributesToClear({
           helper,
-          blackList: excludeAttributes,
+          blackList: excludedAttributes,
```

Comment:
Let's take the opportunity to also change the internals `blacklist` to `excludedAttributes`.